src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -94,10 +93,10 @@ public CRAMIterator(final InputStream inputStream, final ReferenceSource referen
         parser = new ContainerParser(cramHeader.getSamFileHeader());
     }
 
-    public CRAMIterator(final SeekableStream seekableStream, final ReferenceSource referenceSource, final long[] coordinates, final ValidationStringency validationStringency)
+    public CRAMIterator(final SeekableStream seekableStream, final CRAMReferenceSource referenceSource, final long[] coordinates, final ValidationStringency validationStringency)
             throws IOException {
         if (null == referenceSource) {
-            throw new CRAMException("A reference source is required for CRAM files");
+        //    throw new CRAMException("A reference source is required for CRAM files");
```

Comment:
Why did you comment out this line? It seems to me that we still want to throw if `referenceSource` is null.